test/app/wrappedComponent.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { FormattedMessage, injectIntl } from 'react-intl'
+import messages from './defineMessages'
+
+class WrappedComponent extends React.Component<any, any>{
+    render() {
+        const { formatMessage } = this.props.intl;
+
+        const emailPlaceholder = formatMessage({
```

Comment:
How to ensure `formatMessage` is from `'react-intl'` but not from user defined. does this check is safe enough. at least don't break existing code.

Reply:
Never mind.

Reply:
It is injected from the hoc from `injectIntl` from `react-intl`. I did not mean to break existing code but I wasn't sure about the way I implemented this...